.github/workflows/reusable-revised-its.yml

```diff
@@ -57,6 +57,19 @@ on:
       AWS_SECRET_ACCESS_KEY:
         required: false
         type: string
+      BACKWARD_INCOMPATIBILITY_IT_ENABLED:
```

Comment:
I think we can remove this as well. If druid previous version is null, we can just skip running the backward compatibility it.